Set1.txt

```diff
@@ -1,59 +1,64 @@
 Name	Set	ImageFile	Type	Base Class	Top Class	Skill Type	Tier	Health	Cost	Text
-Assassin	Set1	AssassinClass	Class	Rogue			2	20		Ability (2 AP) : Deal 2 damage to an enemy Hero, and give them (Poison 1) or (Bleed 1)
-Chronomancer	Set1	ChonomancerClass	Class	Mage			3	20		Ability (2 AP) : Give an enemy Hero Daze 3'
+Assassin	Set1	AssassinClass	Class	Rogue			2	20		"Ability (2 AP) : Deal 2 damage to an enemy Hero, and give them (Poison 1) or (Bleed 1)"
+Chronomancer	Set1	ChonomancerClass	Class	Mage			3	20		Ability (2 AP) : Give an enemy Hero (Daze 3).
 Cryomancer	Set1	CryomancerClass	Class	Mage			2	20		Ability (2AP) : Deal 1 damage to an enemy Hero and Freeze it.
 Electromancer	Set1	ElectromancerClass	Class	Mage			2	20		Ability (2AP) : Deal 1 damage twice to all enemy characters.
 Mage	Set1	MageClass	Class	Mage			1	20		Ability (1AP) : Deal 1 damage each to up to 2 enemy characters.
-Pyromancer	Set1	PyromancerClass	Class	Mage			2	20		Ability (2AP) : Deal 4 damage, divided as you choose amongst 2 or more enemy characters (max 2 damage per character).
-Rogue	Set1	RogueClass	Class	Rogue			1	20		Ability (1AP) : Draw 1 card, then discard 1 card.
+Pyromancer	Set1	PyromancerClass	Class	Mage			2	20		"Ability (2AP) : Deal 4 damage, divided as you choose amongst 2 or more enemy characters (max 2 damage per character)."
+Rogue	Set1	RogueClass	Class	Rogue			1	20		"Ability (1AP) : Draw 1 card, then discard 1 card."
```

Comment:
Excel leaves quotes. Might want to do a find and replace for quotes (") and replace with nothing (" ")